README.md

```diff
@@ -25,8 +25,8 @@ This project is a playground to play with Kubernetes.
 The provisioning and configuration process has two phases:
 
 1. Prepare a base Vagrant box.
-   1. Provision and configure a VM (`vagrant up base-box-builder.k8s-play.local`).
-   1. Export a Vagrant box based on the `vagrant up base-box-builder.k8s-play.local` VM.
+   1. Provision and configure a VM (`vagrant base-box-builder.k8s-play.local`).
```

Comment:
are you sure that "up" is not needed? if I remember correctly I run:
vagrant up base-box-builder.k8s-play.local
but I have not tried now

Reply:
Hope now it's clearer (added one commit)